src/stages/fillSlot.ts

```diff
@@ -513,13 +552,14 @@ async function runRetryCheck<T, G>(
 async function runRetry<T, G>(
   convoStorageLayer: G,
   getState: () => WolfState,
+  slots: Slot<G>[],
   abilities: Ability<T, G>[],
```

Comment:
Parameter `slots` and `abilities` can be simplified to pass in `flow` which contains both. Simplifies parameter from 2 -> 1